arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -1647,6 +1671,36 @@
         "message"
       ]
     },
+    "AliasPathType": {
+      "properties": {
+        "path": {
+          "type": "string",
+          "description": "Gets or sets the path."
+        },
+        "apiVersions": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the api versions."
```

Comment:
Please don't add "Gets or sets" to the description. The generator will take care of this for you in each languages' way.

Please apply this to other property descriptions.